library/core/src/bool.rs

```diff
@@ -60,4 +60,50 @@ impl bool {
     pub fn then<T, F: FnOnce() -> T>(self, f: F) -> Option<T> {
         if self { Some(f()) } else { None }
     }
+
+    /// Returns either `true_val` or `false_val` depending on the value of
+    /// `condition`, with a hint to the compiler that `condition` is unlikely
```

Comment:
Nit: `condition` isn't a var or arg name here.
```suggestion
    /// `self`, with a hint to the compiler that `self` is unlikely
```
(not sure if `self` is the best choice here either...)